dist/index.html

```diff
@@ -10,50 +10,52 @@
+  <div tabindex="0" class="welcome-user">Welcome</div>
   <hr class="line">
    <section class="interface-wrapper">
       <section class="buttons">
         <div class="control-panel">Control Panel</div>
         <div class="button-side1">
-        <button class="control-button" id="past">Past</button>
-        <button class="control-button" id="present">Present</button>
+        <button class="control-button" tabindex="0" id="past">Past</button>
+        <button class="control-button" tabindex="0" id="present">Current</button>
         </div>
         <div class="button-side2">
-        <button class="control-button" id="upcoming">Upcoming</button>
-        <button class="control-button" id="pending">Pending</button>
+        <button class="control-button" tabindex="0" id="upcoming">Upcoming</button>
+        <button class="control-button" tabindex="0" id="pending">Pending</button>
       </div>
       </section>
       <section class="plan-trip">
-        <form>
-          <div class="plan-header">Plan your next trip!</div><br>
-            <label for="trip-date">Date:</label>
-            <input type="date" id="trip-date" placeholder="YYYY/MM/DD" required /><br>
-            <label for="trip-duration">Duration:</label>
-            <input type="number" id="trip-duration" placeholder="Days" required /><br>
-            <label for="trip-travelers">Number of Travlers:</label>
-            <input type="number" id="trip-travelers" placeholder="Number of Travlers" required /><br>
-            <label for="trip-destination">Choose a Destination:</label>
-              <select id="destination" name="destination">
-                <option value="Peru">Peru</option>
-                <option value="Bahamas">Bahamas</option>
+        <form class="plan-new-trip">
+          <div tabindex="0"class="plan-header">Plan your next trip!</div><br>
+            <label tabindex="0"for="trip-date">Date:</label>
+            <input tabindex="0" type="date" name='date' id="trip-date" placeholder="YYYY/MM/DD" required /><br>
+            <label tabindex="0"for="trip-duration">Duration:</label>
+            <input tabindex="0" type="number" name="duration" id="trip-duration" placeholder="Days" required /><br>
+            <label tabindex="0" for="trip-travelers">Number of Travlers:</label>
+            <input tabindex="0" type="number" name="travelers" id="trip-travelers" placeholder="Number of Travlers" required /><br>
+            <label tabindex="0" for="trip-destination">Choose a Destination:</label>
+              <select id="destination" name="destinate">
               </select><br>
-              <div class="costs">Cost:</div>
-              <div class="costs">Agent Fee:</div>
-            <button type="submit" id="submit-trip" class="disable" disabled >Save</button>
+              <div class="costs">Cost with Agent Fee: </div>
+            <input type="submit" aria-label="submit trip button" value="submit" id="submit-trip" ></button> 
```

Comment:
good use of an ARIA tag here,  a property in plain english. `aria-label="submit trip button"`